.circleci/config.yml

```diff
@@ -2,15 +2,15 @@ version: 2.1
 jobs:
   build:
     machine:
-      image: ubuntu-2004:202101-01
+      image: ubuntu-2204:2022.07.1
     steps:
       - checkout
       - run:
           name: Change owner on project dir (default user = circleci (1001) and docker needs uid 1000)
           command: sudo chown -R 1000:1000 ../project
       - run:
           name: Install PHP dependencies
-          command: docker run -u www-data -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:6.0 php -d memory_limit=4G /usr/local/bin/composer --prefer-dist install
+          command: docker run -u www-data -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:8.1 composer --prefer-dist install
```

Comment:
```suggestion
          command: docker run -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:8.1 composer --prefer-dist install
```

Reply:
Nope because we use docker and not docker-compose here to make a install of the vendor. If we do it composer will install packages with root permission and when the PIM will be launch it would not open the vendor (as it's executed with www-data user)

Reply:
Yes you're right, problem is user option is specified twice 😄

Reply:
Well seen ! :greglelynx: